lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -418,7 +461,10 @@ impl ModuleCodeGenerator<X64FunctionCode, X64ExecutionContext, CodegenError>
         };
 
         let total_size = assembler.get_offset().0;
-        let output = assembler.finalize().unwrap();
+        let _output = assembler.finalize().unwrap();
```

Comment:
The `_` prefix usually means unused, but this is used? Could you rename it? Actually, I expected that `output` would work here? You can `let` the same name twice in rust, using the first one in the initializer for the second.

Reply:
`CodeMemory` isn't supposed to receive a type from dynasm. So there needs to be another statement to copy data from `_output` to `output`.